python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -43,8 +54,8 @@ def __init__(self, driver=None, host=None, port=None, database=None):
                     stored in sims_data/OpSimData/
         """
         if database is None:
```

Comment:
Honestly, I think we should just go all the way and remove this default behavior.  If database=None, the class should print a warning message and do nothing.  You are right: we don't want to encourage people to use the testing database.